replace user and role creation with corresponding modules (#323)
NETWAYS/ansible-collection-elasticstack: docs/role-logstash.md

* *logstash_elasticsearch*: Address of Elasticsearch instance for default output (default: list of Elasticsearch nodes from `elasticsearch` role or `localhost` when used standalone)
* *logstash_security*: Enable X-Security (No default set, but will be activated when in full stack mode)
* *logstash_user*: Name of the user to connect to Elasticsearch (Default: `logstash_writer`)
* *logstash_user_email*: email-address that is linked with the logstash_user (Default: `""`)
* *logstash_user_fullname*: fullname that is linked with the logstash_user (Default: `Internal Logstash User`)

> * *logstash_user_password*: Password of Elasticsearch user. It must be at least 6 characters long (default: `password`)

Reviewer: ```suggestion
* *logstash_user_password*: Password of `logstash_user` in Elasticsearch. It must be at least 6 characters long (default: `password`)
```
I know, this was already wrong before the change. But please use the chance to fix it.

Reviewer 2: Since the user name isnt hardcoded anymore I would not mention the previous hardcoded username in the doc. I would go with something like "Password of logstash user in Elasticsearch".
Mentioning the `logstash_user` inside of the doc could be misleading if you dont use the default name.

* *logstash_role_name*: Name of the logstash role that is getting created (Default: `logstash_writer`)
* *logstash_role_cluster_privileges*: Cluster privileges the role has access to (default: `"manage_index_templates", "monitor", "manage_ilm"`)
* *logstash_role_indicies_names*: Indices the role has access to (default: `"ecs-logstash*", "logstash*", "logs*"`)

> * *logstash_role_indicies_privileges*: Indices the role has access to (default: `"write", "create", "delete", "create_index", "manage", "manage_ilm"`)

Reviewer: ```suggestion
* *logstash_role_indicies_privileges*: Index permissions the role has on `logstash_role_indicies_names`: (default: `"write", "create", "delete", "create_index", "manage", "manage_ilm"`)
```
I guess, that was just a copy&paste error.
